Replicator/tests/SG.cc

```diff
@@ -138,9 +138,37 @@ bool SG::deleteUser(const string &username) const {
 }
 
 bool SG::assignUserChannel(const std::string& username, const std::vector<std::string> &channelIDs) const {
```

Comment:
We probably need an argument telling what collections will be involved. Or we may want to test different channelIDs for different collections,

Reply:
I believe the commit I just made should address those two concerns